### DISCUSSION TITLE
Prism library license change

### DISCUSSION THREAD

**Discussion by zarlo** [AvaloniaCommunity/Prism.Avalonia#100] [General]
with prism library license change i think you should make note of it in the read me as people may have to pay for a license to use prism library

**Comment by DamianSuess** [comment]
Good call, @zarlo.

Since they did recently update to a split-use licensing structure, "_... MIT or a Prism Library Commercial Use License.._" during the 9.x version (_currently unreleased_).

Since I currently have Prism.Avalonia set up under the MIT license and we're using a version of Prism.Core under the same (v8.x), it's currently not an issue. However, moving forward into v9., this will need to be updated. I'll reach out to Dan and Brian to cover our bases.

Thank you again for bringing this up.

**Comment by baochenw** [comment]
I see that there have already been two 9.0 preview releases, both licensed under MIT. Does this mean that the licensing issue has been resolved?

**Reply by zarlo** [reply]
Prism.Avalonia is under MIT but Prism is not (Prism can be licensed either under the Prism Community License or the Prism Commercial license.)

**Reply by DamianSuess** [reply]
I'm already in discussions with the guys on Prism (Dan/Brian). Overall, we're looking out for the best interest of the community.  If it wasn't for the open source license, I too wouldn't have been able to reach my customers in ways which I have. We got a great community over here, and looking forward to serving you all in the best way possible :)

**Comment by DamianSuess** [comment]
This project will be hosted as part of the Prism Library, and all future development will take place there.

I've been working with, Dan, on getting the repository transitioned to continue development of the project. Overall this is a wonderful step to keep our project moving forward with full integration and making it official.
